src/l2d/useL2d/main.ts

```diff
@@ -47,8 +47,19 @@ export function releaseL2d() {
   LAppDelegate.releaseInstance();
 }
 
-export function loadL2dAsset(ResourcesPath: string, ModelDir: string, center?: {x: number, y: number}, offDefaultMove?: boolean) {
+export function loadL2dAsset(ResourcesPath: string, ModelDir: string, loadData?: {x: number, y: number, zoom: number}, offDefaultMove?: boolean) {
+  const center = loadData && {
+    x: loadData.x,
+    y: loadData.y
+  };
+
   LAppLive2DManager.getInstance().changeScene(ResourcesPath, ModelDir, center, offDefaultMove);
+  if(loadData) {
+    LAppDelegate.getInstance()._view._viewMatrix.scale(
+      LAppDefine.ViewScale * loadData.zoom,
+      LAppDefine.ViewScale * loadData.zoom,
+    );
+  }
```

Comment:
중심이 제대로 설정안되는 이슈때문이라면 아래처럼 코드를 해주면 동작하는 것 같아요

```suggestion
  const center = loadData && {
    x: loadData.x,
    y: loadData.y
  };

  LAppLive2DManager.getInstance().changeScene(ResourcesPath, ModelDir, center, offDefaultMove);
  if (loadData) {
    setZoom(loadData.zoom, loadData.x, loadData.y)
  }
```